pandas/core/arrays/base.py

```diff
@@ -1213,7 +1237,7 @@ def _formatter(self, boxed: bool = False) -> Callable[[Any], Optional[str]]:
     # Reshaping
     # ------------------------------------------------------------------------
 
-    def transpose(self, *axes) -> ExtensionArray:
+    def transpose(self, *axes: int) -> ExtensionArray:
```

Comment:
is this a standard pattern?  i have no idea off the top of `*args` are supposed to be annotated

Reply:
It's for compatibility with numpy, and that's what they use there.